### ISSUE TITLE
Improve appearance of confirmation message displayed when adding an add-on to a collection

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-frontend#5011]
The current implementation looks like this:

<img width="489" alt="screenshot 2018-05-15 16 18 44" src="https://user-images.githubusercontent.com/142755/40082703-c4d7f63e-585f-11e8-8c17-aab80c19ffb1.png">

I think it would be better if the message was closer to the "Search" select box, and also @pwalm requested that the message fade out after 5 seconds.

**Comment by bobsilverberg** [comment]
Note, this is a follow-up to issue #5009 so the message will not exist until that change lands.

**Comment by pwalm** [comment]
Can we move the green message bar and the search field slightly (~10px) closer, please?

**Comment by willdurand** [comment]
Quick question: would it make sense to pu the green message bar below the label?

**Comment by pwalm** [comment]
Nah, we shouldn't split the label from the field.

**Comment by ioanarusiczki** [comment]
Verified fixed on AMO dev with FF60(Win10 & Android 8.0)
@bobsilverberg 1.The message fades away approximately in 8 seconds. I remember some discussions about 5 seconds or so.  
2. A scenario I tested is when an extension is added and while the message is displayed I removed another extension that already existed in the collection -> added to collection message is still there and no other message follows for the one removed. 

![messageincollections](https://user-images.githubusercontent.com/33448286/40916669-355e6f42-6809-11e8-9f88-3124dd0d7857.gif)

**Comment by bobsilverberg** [comment]
Thanks @ioanarusiczki. I just tested locally, and it only seems to be displayed for 5 seconds for me.

As for removing an add-on after the message is displayed, I think it's basically okay. We do not display a message to confirm that an add-on was removed, and because the confirmation goes away after 5 seconds I think it's fine that we do not force-remove it after a user removes an add-on from the collection.
